src/containers/ReviewsIWrote/index.js

```diff
@@ -11,19 +11,30 @@ class ReviewsIWrote extends Component {
   componentDidMount() {
     this.props.readReviewsIWrote()
   }
+  
+  loadMore = () => {
+    if (!this.props.loading) this.props.readReviewsIWrote(false)
+  }
 
   render() {
-    const { reviews, editing, loading, error } = this.props
-
-    return <Reviews reviews={reviews} loading={loading} error={error} editing={editing} />
+    const { reviews, loading, loadingPage, canLoadMore, error } = this.props
+
+    return <Reviews
```

Comment:
We discussed this quite some time ago now, but we generally agreed to use this syntax for multi-line components returned:
```
return (
  <elem>
    ...
  </elem>
)
```